.github/workflows/live-links.yml

```diff
@@ -39,8 +39,10 @@ jobs:
             https://ubuntu.com/static/css/*
             https://www.xilinx.com/applications/*
             https://player.vimeo.com/video/*
+            https://beian.miit.gov.cn/*
             ^https://assets\.ubuntu\.com/?$
             \.css
+            \.js
```

Comment:
Why was .js excluded?

I ran the link-checker locally on prod URL and it flagged  https://ubuntu.com/static/js/dist/release-chart.js
which doesn't exist but  https://ubuntu.com/static/js/dist/release-chart-manager.js does

Reply:
Also this file
https://github.com/canonical/cn.ubuntu.com/blob/244dfa4e0fa44d0f7cbb03e0bb1434261a86583f/templates/blog/newsletter-form.html#L49
doesn't seem to exist

Reply:
Good catch, I have update my PR accordingly